db-commons/src/main/resources/reference.conf

```diff
@@ -117,6 +117,22 @@ bitcoin-s {
         rpcbind = "127.0.0.1"
     }
 
+  dlc = ${bitcoin-s.dbDefault}
+  dlc {
+    # this config key is read by Slick
+    db {
+      name = dlcdb
+      url = "jdbc:sqlite:"${bitcoin-s.dlc.db.path}${bitcoin-s.dlc.db.name}
+    }
+    # PostgreSQL example:
+    # db {
+    #   url = "jdbc:postgresql://localhost:5432/dlc"
```

Comment:
shouldn't this be `dlcdb`?

Reply:
We actually don't need the url anymore, fixed this